calc/main.py

```diff
@@ -6,8 +9,13 @@ def subtract(a, b):
     return a - b
 
 
-def mult(a, b):
-    return a * b
+def mult(*args: float) -> float:
```

Comment:
Waarom geen int | float in de type hinting?

Reply:
Ik denk dat je het best alleen `float` kan gebruiken. `int` valt daar binnen. https://stackoverflow.com/questions/50928592/mypy-type-hint-unionfloat-int-is-there-a-number-type